pkg/crc/machine/bundle/repository.go

```diff
@@ -39,26 +39,47 @@ func (bundle *CrcBundleInfo) readBundleInfo() error {
 	return nil
 }
 
-func GetCachedBundleInfo(bundleName string) (*CrcBundleInfo, error) {
+type Repository struct {
+	CacheDir string
+	OcBinDir string
+}
+
+func (r *Repository) Get(bundleName string) (*CrcBundleInfo, error) {
 	var bundleInfo CrcBundleInfo
-	bundleInfo.cachedPath = getCachedBundlePath(bundleName)
+	bundleInfo.cachedPath = getCachedBundlePath(r.CacheDir, bundleName)
 	if !bundleInfo.isCached() {
-		return nil, fmt.Errorf("Could not find cached bundle info")
+		return nil, fmt.Errorf("could not find cached bundle info in %s", bundleInfo.cachedPath)
 	}
 	err := bundleInfo.readBundleInfo()
 	if err != nil {
 		return nil, err
 	}
-	if err := bundleInfo.installSymlinkOrCopy(); err != nil {
+	return &bundleInfo, nil
+}
+
+func (r *Repository) Use(bundleName string) (*CrcBundleInfo, error) {
+	bundleInfo, err := r.Get(bundleName)
+	if err != nil {
+		return nil, err
+	}
+	if err := bundleInfo.installSymlinkOrCopy(r.OcBinDir); err != nil {
```

Comment:
This is pre-existing, but this method name is missing a hint that it is about the `oc` binary.

Reply:
done